Cargo.toml

```diff
@@ -24,6 +24,12 @@ license = "Apache-2.0"
 edition = "2024"
 rust-version = "1.88.0" # Attention - Change the MSRV in rust-toolchain and in .circleci/config.yml as well
 
+[package.metadata.cargo-semver-checks.lints]
```

Comment:
Please remove this section. We should not have to worry about two places for updating CI, only the BASELISE_REV should be needed to be updated. And if you disagree we should discuss.